src/routes/welcome/start/index.tsx

```diff
@@ -58,56 +90,31 @@ export default function Start() {
           <ButtonV2
             fullWidth
             onClick={() =>
-              setLocation(page === 3 ? "/generate/1" : `/start/${page + 1}`)
+              navigate(page === 3 ? "/generate/1" : `/start/${page + 1}`)
             }
           >
             {browser.i18n.getMessage("next")}
             <ArrowRightIcon style={{ marginLeft: "5px" }} />
           </ButtonV2>
-          <ButtonV2
-            secondary
-            fullWidth
-            onClick={() => setLocation("/generate/1")}
-          >
+          <ButtonV2 secondary fullWidth onClick={() => navigate("/generate/1")}>
             {browser.i18n.getMessage("skip")}
           </ButtonV2>
         </ButtonWrapper>
       </ExplainerSection>
       <Pagination>
-        {Array(3)
-          .fill("")
-          .map((_, i) => (
-            <Page
-              onClick={() => setLocation(`/start/${i + 1}`)}
-              key={i}
-              active={page === i + 1}
-            />
-          ))}
+        {Views.map((_, i) => (
+          <Page
+            key={i}
+            active={page === i + 1}
+            onClick={() => navigate(`/start/${i + 1}`)}
```

Comment:
This could/should be replaced with `navigate(i+1)`. See https://github.com/arconnectio/ArConnect/issues/568.